### ISSUE TITLE
Add aria-hidden="true" to presentational tables

### ISSUE THREAD

**Issue by TedGoas** [emailmonday/Cerberus#101]
Add `aria-hidden="true"` to presentational tables as a companion to `role="presentation"`. [reference](https://litmus.com/blog/ultimate-guide-accessible-emails).

**Comment by kmedeiros** [comment]
Hi @TedGoas , 

I was curious about the recent addition of `aria-hidden="true"` to presentation tables and was hoping you could provide some clarification.

On the [Litmus reference](https://litmus.com/blog/ultimate-guide-accessible-emails) it states:

> In addition to role=”presentation,” aria-hidden=”true” is another HTML attribute that can be added to elements in your HTML that are for visual content and should be hidden from screen readers.

I'm having trouble understanding because the tables within the templates contain a mix of visual content _and_ text that should be human- and machine-readable.

The [W3C documentation](https://www.w3.org/TR/wai-aria/states_and_properties#aria-hidden) states:

> Authors MAY, with caution, use aria-hidden to hide visibly rendered content from assistive technologies only if the act of hiding this content is intended to improve the experience for users of assistive technologies by removing redundant or extraneous content. Authors using aria-hidden to hide visible content from screen readers MUST ensure that identical or equivalent meaning and functionality is exposed to assistive technologies.

All the tables in the hybrid template were updated to reflect this change. Does this mean the addition of `aria-hidden="true"` will cause the text within the `<td></td>` tags to not be visible to assistive devices?

Thanks for your help. I appreciate the accessibility improvements! 👍

**Comment by TedGoas** [comment]
@kmedeiros To the best of my knowledge, this change is meant to hide the layout `<table>`s from being read by screen readers since they do not contain data and are meant purely for layout. `<table>`s that contain data sets _should not_ have `aria-hidden="true"` attributes since they should be read as data by screen readers, but all `<table>`s (so far) in the Cerberus are used purely for layout rather than data.

I believe the content within the `<table>`s should still be read, and I have [another issue](https://github.com/TedGoas/Cerberus/issues/102) to add more semantic tags around content. However (confession) I haven't thoroughly tested this and would welcome any feedback you have regarding how the latest templates _actually display_ in assistive technologies. I will gladly re-open this or create a new issue to improve this further!

**Comment by kmedeiros** [comment]
I'm planning to work with our university's disability office to test this in NVDA and JAWS. I'll post details soon.
